### ISSUE TITLE
Migration Error 2.0.8 to 2.0.9 update

### ISSUE THREAD

**Issue by davidhellmann** [verbb/navigation#323]
### Describe the bug

Get an error when I try to update the plugin from 2.0.8 to 2.0.9

### Steps to reproduce

1. hit the update button 🚀 

### Craft CMS version

4.3.0

### Plugin version

2.0.8

### Multi-site?

No

### Additional context

```php
Unknown Property: Setting unknown property: verbb\navigation\records\Nav::maxNodesSettings

Migration: verbb\navigation\migrations\m221025_000000_propagation

Output:

Exception: Setting unknown property: verbb\navigation\records\Nav::maxNodesSettings (/var/www/html/vendor/yiisoft/yii2/base/Component.php:209)
#0 /var/www/html/vendor/yiisoft/yii2/db/BaseActiveRecord.php(322): yii\base\Component->__set('maxNodesSetting...', Array)
#1 /var/www/html/vendor/craftcms/cms/src/db/ActiveRecord.php(56): yii\db\BaseActiveRecord->__set('maxNodesSetting...', Array)
#2 /var/www/html/vendor/verbb/navigation/src/services/Navs.php(215): craft\db\ActiveRecord->__set('maxNodesSetting...', Array)
#3 /var/www/html/vendor/craftcms/cms/src/services/ProjectConfig.php(1125): verbb\navigation\services\Navs->handleChangedNav(Object(craft\events\ConfigEvent))
#4 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
#5 /var/www/html/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
#6 /var/www/html/vendor/craftcms/cms/src/models/ProjectConfigData.php(82): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent))
#7 /var/www/html/vendor/craftcms/cms/src/services/ProjectConfig.php(515): craft\models\ProjectConfigData->commitChanges(Array, Array, 'navigation.navs...', true, NULL, true)
#8 /var/www/html/vendor/craftcms/cms/src/services/ProjectConfig.php(475): craft\services\ProjectConfig->_setInternal('navigation.navs...', Array, NULL, true, false)
#9 /var/www/html/vendor/verbb/navigation/src/migrations/m221025_000000_propagation.php(53): craft\services\ProjectConfig->set('navigation.navs...', Array)
#10 /var/www/html/vendor/craftcms/cms/src/db/Migration.php(49): verbb\navigation\migrations\m221025_000000_propagation->safeUp()
#11 /var/www/html/vendor/craftcms/cms/src/db/MigrationManager.php(233): craft\db\Migration->up(true)
#12 /var/www/html/vendor/craftcms/cms/src/db/MigrationManager.php(149): craft\db\MigrationManager->migrateUp(Object(verbb\navigation\migrations\m221025_000000_propagation))
#13 /var/www/html/vendor/craftcms/cms/src/services/Updates.php(252): craft\db\MigrationManager->up()
#14 /var/www/html/vendor/craftcms/cms/src/controllers/BaseUpdaterController.php(493): craft\services\Updates->runMigrations(Array)
#15 /var/www/html/vendor/craftcms/cms/src/controllers/UpdaterController.php(203): craft\controllers\BaseUpdaterController->runMigrations(Array, 'restore-db')
#16 [internal function]: craft\controllers\UpdaterController->actionMigrate()
#17 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#18 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#19 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('migrate', Array)
#20 /var/www/html/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('updater/migrate', Array)
#21 /var/www/html/vendor/craftcms/cms/src/web/Application.php(712): craft\web\Application->runAction('updater/migrate')
#22 /var/www/html/vendor/craftcms/cms/src/web/Application.php(258): craft\web\Application->_processUpdateLogic(Object(craft\web\Request))
#23 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#24 /var/www/html/web/index.php(21): yii\base\Application->run()
#25 {main}
```

**Comment by engram-design** [comment]
Should be fixed if you're able to double check. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [2.0.10](https://github.com/verbb/navigation/releases/tag/2.0.10)
